{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/__init__.py

```diff
@@ -28,4 +29,5 @@ class UnsupportedPythonError(Exception):
     pass
 {%- else %}
     from .example_mod import *   # noqa
+    __all__ += .example_mod.__all__
```

Comment:
I don't really like this. Why not do:
```suggestion
    __all__ += ['primes', 'do_primes']
```
to be **really** explicit?

Reply:
because in this case one doesn't have to remember to go back to the top level, but declare stuff public only where they are defined. 
At least I think that's the logic, copied from here:
https://github.com/numpy/numpy/blob/master/numpy/lib/__init__.py

Reply:
I can see the argument either way, but it just feels a little cludgy.

Also I don't consider numpy a great example of namespace management :stuck_out_tongue_closed_eyes: :stuck_out_tongue_closed_eyes:

Reply:
Well, it's better than what we have atm. 
I think your proposal would work nicely in smaller subpackages, but when an `__all__` is already many lines long (see astropy.stats.funcs from where `*` is imported to astropy.stats along with the rest of the subsubpackages in stats) it can quickly become a mess to manage.

Reply:
The thing with this is that it's an example for people to modify, so I was erring on the side of understandable rather than extensible.

Reply:
that's a valid point. then I'll add some comments.